src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -351,16 +338,15 @@ public void start(Map<String, String> props) {
             isDlqEnabled = false;
             dlqTopicName = null;
         }
-        
+
         topicsToIngestionProps = getTopicsToIngestionProps(config);
-        
+
         // this should be read properly from settings
         kustoIngestClient = createKustoIngestClient(config);
-        
-        log.info(String.format("Started KustoSinkTask with target cluster: (%s), source topics: (%s)", 
-            url, topicsToIngestionProps.keySet().toString()));
+
+        log.info("Started KustoSinkTask with target cluster: ({}), source topics: ({})", url, topicsToIngestionProps.keySet());
```

Comment:
just out of curiosity - why did you drop the explicit call for tostring

Reply:
It's already called by the log Formatter.